doc/us-en/tutorials/Device-connect-to-OpenEdge-with-hub-module.md

```diff
@@ -3,6 +3,7 @@
 **Statement**:
 
 - The device system used in this test is Darwin
+- This test is based on the OpenEdge you downloaded in [Quick Install](../setup/Quick-Install), and import the default configuration package
```

Comment:
It should be installed for OpenEdge when you read this document, more details please refer to [How-to-quick-install-OpenEdge](../setup/Quick-Install.md)